File: Pertemuan4.1-Java/Matriks.java
class Matriks {
    private int baris;
    private int kolom;
    private int mat[][];

    Matriks() {

    }

    Matriks(int baris, int kolom) {
        this.baris = baris;
        this.kolom = kolom;
        mat = new int[baris][kolom];
    }

    void setBaris(int baris) {
        this.baris = baris;
    }

    int getBaris() {
        return baris;
    }

    void setKolom(int kolom) {
        this.kolom = kolom;
    }

    int getKolom() {
        return kolom;
    }

    void setMat() {
        mat = new int[baris][kolom];
    }

    int[][] getMat() {
        return mat;
    }

    void setSel(int baris, int kolom, int nilai) {
        mat[baris][kolom] = nilai;
    }

    int getSel(int baris, int kolom) {
        return mat[baris][kolom];
    }

}

File: Pertemuan4.1-PHP/Matriks.php
<?php
class Matriks
{
    private $baris;
    private $kolom;
    private $mat;

    function __construct()
    {
    }

    function setBaris($baris)
    {
        $this->baris = $baris;
    }

    function getBaris()
    {
        return $this->baris;
    }

    function setKolom($kolom)
    {
        $this->kolom = $kolom;
    }

    function getKolom()
    {
        return $this->kolom;
    }

    function setMat($mat)
    {
        $this->mat = $mat;
    }

    function getMat()
    {
        return $this->mat;
    }

    function setSel($baris, $kolom, $nilai)
    {
        $this->mat[$baris][$kolom] = $nilai;
    }

    function getSel($baris, $kolom)
    {
        return $this->mat[$baris][$kolom];
    }
}
